poller/poller.go

```diff
@@ -1,42 +1,80 @@
 package poller
 
 import (
+	"sync/atomic"
 	"time"
+	"unsafe"
 
 	"github.com/godaddy/split-go-serializer/api"
+	"github.com/splitio/go-client/splitio/service/dtos"
 )
 
-// Poller contains cache data and SplitioAPIBinding
+// Poller contains cache data, splitioDataGetter, and required info to interact with aplitio api
 type Poller struct {
-	Cache              int
```

Comment:
I removed the Cache from this struct because client will now use `poller.GetCache()` to get the latest Cache